go-chaos/worker/chaos_worker.go

```diff
@@ -40,10 +40,29 @@ type ChaosProvider struct {
 
 type AuthenticationProvider struct {
 	Audience         string
-	AuthorizationUrl string
+	AuthorizationURL string
 	ClientId         string
 	ClientSecret     string
-	ContactPoint     string
+	// ContactPoint     string
```

Comment:
👍🏼 We can remove it completelty - we dont need it as we port-forward and directly connect to the gatway/broker.